exhale/graph.py

```diff
@@ -615,14 +629,24 @@ def inFileHierarchy(self):
                     return True
         return False
 
-    def inHierarchy(self, classView):
-        if classView:
+    def inHierarchy(self, classType):
```

Comment:
Lets go ahead and rename this to just `hierarchy` or `hierarchyType` for the second parameter (dealers choice).  Probably make this an enum once 2.7 is ditched, but for now we'll work with strings :slightly_smiling_face:

Reply:
Done in https://github.com/svenevs/exhale/pull/114/commits/674aa4be1f70b79560f741df18599da7d715fa36.